recipe/meta.yaml

```diff
@@ -8,9 +8,15 @@ source:
   fn: PyGithub-{{ version }}.tar.gz
   url: https://github.com/PyGithub/PyGithub/archive/v{{ version }}.tar.gz
   sha256: 862f481d5a51825f1f7abc76d3a17e571f378d6b95c629c581f7eccd3ea04200
+  patches:
+    # Adds a patch that provides the ability to create and modify repo content.
+    # Link to the pull request is shown below.
+    #
+    # ( https://github.com/PyGithub/PyGithub/pull/379 )
+    - PR_379.diff
```

Comment:
I understand. I know sometimes we use patches, but they are primarily bug fixes. This goes a bit beyond that by actually adding new functionality. That being said, this was the easiest way forward. Let me think about this some more.